### ISSUE TITLE
Classes in global block should be ignored

### ISSUE THREAD

**Issue by atfzl** [atfzl/eslint-plugin-css-modules#5]
classes defined in global block are being used for linting. These should be ignored.

```scss
.foo {
  /* this class should be used */
}

:global {
  /* all the classes in this block should be ignored */
  .bar {
   }

  .baz {
  }
}
```

**Comment by eddiemoore** [comment]
Also can add other way of defining `:global`
```css
:global .bar {
}
:global .baz {
}
```
At the moment it only ignoring if you define as
```css
:global(.bar) {
}
:global(.baz) {
}
```

**Comment by atfzl** [comment]
Fixed in 2.5.0
